lib/langchain/chunker/base.rb

```diff
@@ -11,5 +11,17 @@ module Chunker
     # - {Langchain::Chunker::Text}
     class Base
     end
+    
+    class RecursiveTextSplitter < Base
```

Comment:
could you put this class into lib/langchain/chunker ,just like the other Chunker Classes do

Reply:
could you put this class into lib/langchain/chunker ,just like the other Chunker Classes do